cmake/f3dPlugin.cmake

```diff
@@ -144,12 +146,13 @@ The `NAME` argument is required. The arguments are as follows:
   * `VTK_MODULES`: The list of VTK modules used by the plugin to link with.
   * `ADDITIONAL_RPATHS`: The list of additional RPATH for the installed binaries on Unix. VTK path is added automatically.
   * `MIMETYPE_XML_FILES`: The list of mimetype files to install. It's useful for file association on OS using Freedesktop specifications.
+  * `CONFIGURATION_DIRS`: The list of configuration directories to install. Generally contain a load-plugins option and format specific options.
```

Comment:
What about having `APPLICATION_CONFIG_FILE` and `THUMBNAIL_CONFIG_FILE` instead? It would make it simpler for the user I suppose by letting him put the json file where he wants and reuse the same json file for app and thumbnail.

Reply:
What if a someone want to use another config ? They need to install it manually ?

Reply:
You mean if we want to support `--config myConfig` stem?

Reply:
of course

Reply:
we already do of course.

Reply:
ok then